tests/Integration/page_objects/search_object.py

```diff
@@ -0,0 +1,19 @@
+from re import search
```

Comment:
While editing and deleting of article , first it will search particular article by using word(ex : python) in search text field then it will perform edit or delete that article